.github/workflows/ci-cd.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
   audit:
     runs-on: ubuntu-latest
-    needs: [bundle_install]
+    needs: [bundle_install, yarn_install]
```

Comment:
Audit should wait until both dependencies are installed first, or else it will grab the wrong cache to use.